### ISSUE TITLE
Support for vector graphic images

### ISSUE THREAD

**Issue by wrighteap** [ONLYOFFICE/DesktopEditors#780]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my idea.

### Your idea.

Support for images in formats such as pdf and svg would be greatly appreciated!

**Comment by ShockwaveNN** [comment]
Images in pdf?

Agree about svg (and I think we already got issue about this), but I don't think anyone store images in pdf

**Comment by wrighteap** [comment]
Yes, svg support is request #102 , and will be a very welcome addition.

I opened this request for vector graphic images, particularly with eps and pdf,  because these formats are relatively commonplace in the scientific community (particularly Maths and Physics), where they are often with latex templates for scientific articles. It would be a useful nicety to have the ability to directly insert eps and pdf images into presentations, to avoid an intermediate conversion step.

**Comment by ShockwaveNN** [comment]
Could you attach both of those example files (eps and pdf)?

**Comment by wrighteap** [comment]
Sure. Here is an example of each format.
[test.zip](https://github.com/ONLYOFFICE/DesktopEditors/files/7703758/test.zip)
.

**Comment by TatianaKirillova** [comment]
Hello, **@wrighteap**
Thanks for your request!
I can confirm that the problem exists. I created ticket 54599 in our private  tracker with your proposal.
Problem reproduces in Document Server.

**Comment by xyont** [comment]
in Document i have successfully insert vector image (wmf) even the application restrict only bitmap format is allowed.

![2022-02-15 08_26_19-ONLYOFFICE Desktop Editors](https://user-images.githubusercontent.com/72367311/154164700-04bffa8c-aee4-4edf-860b-7310b02b5945.jpg)

editing, reopening the document is working without any problems. however saving in PDF is not as expected, the quality image are blocked by black colors.


![2022-02-15 08_27_01-DocumentContainsVectorImage pdf - Adobe Reader](https://user-images.githubusercontent.com/72367311/154164734-5bbb6df9-35be-4e46-b14d-f27a3ff152e5.jpg)

high quality document are essential for technical writer and many others, supporting to one of these file type could be great features to improves.

**Comment by ShockwaveNN** [comment]
@xyont Could you create a new issue, because seems this is another problem (we got several problems with wmf) since original feature request was about inserting eps and pdf

**Comment by sigreer** [comment]
We'd like to be able to print documents on letterheaded templates directly from OO rather than ordering expensive stationery sets. It's more economical for us and leads to much less tree wastage. Just to add to some to the conversation regarding formats:
PDF, AI and EPS are the common output formats for Adobe Illustrator with EPS being the established standard for vectors where print is concerned. SVG is a web format but I suspect probably the most sought after due to the availability of images, icon packs etc. online.

Just my two cents, but if you had to pick the 'essentials' to offer compatibility for, I'd go with EPS and SVG.
